src/Symfony/Component/Debug/Tests/FatalErrorHandler/ClassNotFoundFatalErrorHandlerTest.php

```diff
@@ -35,12 +35,11 @@ public function testHandleClassNotFound($error, $translatedMessage)
     }
 
     /**
+     * @group legacy
```

Comment:
This is the first time we have to use this annotation: this tests is now excluded from being run with components=high because 3.0 removed a class that is used in the test